geopandas/explore.py

```diff
@@ -468,7 +468,7 @@ def _colormap_helper(_cmap, n_resample=None, idx=None):
                 color = np.apply_along_axis(
                     colors.to_hex,
                     1,
-                    _colormap_helper(cmap, n_resample=k, idx=binning.yb),
+                    _colormap_helper(cmap, n_resample=binning.k, idx=binning.yb),
```

Comment:
I assume the answer to this is yes, but thought I'd check. Can we rely on `binning.k` being defined? It doesn't seem to explicitly be part of the `MapClassifier` base class interface - it looks like it is always defined, just sometimes as part of `_set_bins` sometimes in the init, sometime implicitly from arguments in the init.

Reply:
Yes, it is always defined. But this is a good point, I'll make a test for this over in mapclassify.

Reply:
It actually is already implicitly tested here https://github.com/pysal/mapclassify/blob/e3ce03c6028ddc855d2f81227f67181eabeee435/mapclassify/tests/test_classify.py#L9